lib/membrane/parent_spec.ex

```diff
@@ -123,6 +123,40 @@ defmodule Membrane.ParentSpec do
   clock. The pipeline clock is the default clock used by elements' timers.
   For more information see `Membrane.Element.Base.def_clock/1`.
 
+  # Crash Groups
+  Crash group is a logical entity that prevents the whole pipeline from crashing when one of its children crashes.
+  Internally crash groups utilizes process monitoring mechanism to detect child crash and thanks to that Membrane
+  doesn't influence value of `:trap_exit` flag.
+
+  ## Adding children to the Crash Group
+
+  ```elixir
+  children = %{
+    {:endpoint_bin, endpoint_id} => %EndpointBin{
+      # ...
+    }
+  }
+
+  spec = %ParentSpec{children: children, crash_group: {endpoint_id, :temporary}}
+  ```
+
+  Crash group is defined by two element tuple, first element is group id which is of type `Membrane.Crashgroup.name_t()`, and the second is a mode. Currently we support only
+  `:temporary` which means that membrane will not make any attempts to restart crashed child.
+
+  In above snippet we create a new children - EndpointBin and we add it to crash group with id `endpoint_id`. When EndpointBin crashes the whole pipeline will still be alive.
```

Comment:
use a tool like grammarly to check the spelling for you

```suggestion
  In the above snippet, we create new children - EndpointBin and we add it to the crash group with id `endpoint_id`. When EndpointBin crashes, the rest of the pipeline will still be alive.
```